### PR TITLE
[EN] Creates install lesson for phoenix

### PR DESCRIPTION
**Pull request by jorgedjr21**
Creates the first topic for phoenix framework lessons

https://github.com/elixirschool/elixirschool/issues/1602

### CODE REVIEW

File: lessons/en/phoenix/installing.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  """
+}
+---
+
+## Getting Started
+
+### Some considerations
+
+Before aaaaaaaaaaaaaaa creating a Phoenix project, you will need to install elixir in your machine. The latest version of phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
+
```
Comment (doomspork) [inline lessons/en/phoenix/installing.md:15]:
```suggestion
Before creating a new Phoenix app we need to ensure we have compatible versions of Elixir installed. The latest version of Phoenix framework needs [Elixir 12 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later) and [Erlang 22 or later](https://hexdocs.pm/phoenix/installation.html#erlang-22-or-later).
```

In general we prefer to use "we" in places of "you" or "I" so it the learning process feels collaborative and less directed at someone or about someone. Could you work through the lesson updating those references?

File: lessons/en/phoenix/installing.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+You will also need some database to run your application,  if you are just getting started with Phoenix, we recommend you to install PostgreSQL. You can check the [installation guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides) to install it to your OS.
+
```
Comment (doomspork) [inline lessons/en/phoenix/installing.md:17]:
```suggestion
In addition we'll need a database to run our application, we recommend using PostgreSQL. You can check the [installation guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides) to install it to your OS.

```

**Review by doomspork — CHANGES_REQUESTED** [review]
Thanks again for putting this together @jorgedjr21! Before we spend too much time reviewing this could you please update things to reflect our preferred usage of "we" instead of "you". I left a couple suggestions to help get you started.

Let me know if you have any questions!

@SophieDeBenedetto would you please review this lesson as well? I think you had a better vision for the Phoenix content.

File: lessons/en/phoenix/installing.md (hunk 1 above)
Comment (jorgedjr21) [reply lessons/en/phoenix/installing.md:15]: Yeah, thanks for reviewing the PR @doomspork . I will update the PR next weekend!

**Review by doomspork — APPROVED** [review]
Thank you @jorgedjr21 🎉

**Comment by vbrazo** [comment]
Thanks, @jorgedjr21!

Welcome to our community!

**Pull request merged**